src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Value.ReadTests.cs

```diff
@@ -488,6 +488,11 @@ private static void DeserializeLongJsonString(int stringLength)
         }
 
         [Theory]
+        [InlineData("1:00:00")]
+        [InlineData("1")]
+        [InlineData("10")]
+        [InlineData("00:01")]
+        [InlineData("0:00:02")]
```

Comment:
Does the change allow for any new formats that specify milliseconds?

Reply:
I added a few test cases although some of that is quite likely already covered by stuff like 23:59:59.9999999